.circleci/config.yml

```diff
@@ -234,7 +234,9 @@ jobs:
   trigger-beta-build:
     executor: node-browsers-medium-plus
     steps:
-      - checkout
+      - run:
+          name: Shallow git clone
+          command: git clone --depth 1 https://github.com/MetaMask/metamask-extension --branch "$CIRCLE_BRANCH" .
```

Comment:
Update: `CIRCLE_REPOSITORY_URL` is `git@github.com:MetaMask/metamask-extension.git`
When using the git protocol instead of https, github.com is not in the known hosts

I tried this suggestion
`ssh-keyscan github.com >> ~/.ssh/known_hosts`
But got this result
`/bin/bash: /home/circleci/.ssh/known_hosts: No such file or directory`

Reply:
I figured something out.  There's an orb called `guitarrapc/git-shallow-clone`, but it doesn't take the shortest possible path, because it does a `git clone` on the default branch, then a `git fetch --depth 10` on $CIRCLE_BRANCH, then a `git checkout` on $CIRCLE_BRANCH.  All that is unnecessary if you just want the tip of $CIRCLE_BRANCH.  So I wrote an alias to add the SSH key, then clone just one branch.

Reply:
Nice! Tested it on a fork here: https://github.com/MetaMask/metamask-extension/pull/18720
Seems to work